src/diffusers/models/attention_processor.py

```diff
@@ -291,6 +311,29 @@ def prepare_attention_mask(self, attention_mask, target_length, batch_size=None)
             attention_mask = attention_mask.repeat_interleave(head_size, dim=0)
         return attention_mask
 
+    def prepare_encoder_hidden_states(self, hidden_states, encoder_hidden_states=None):
```

Comment:
not a huge fan of the naming here. Can we call it maybe instead:

```suggestion
    def norm_encoder_hidden_states(self, hidden_states, encoder_hidden_states=None):
```

Reply:
renamed here and moved to the check on if encoder_hidden_states is None back to the attention processors to better reflect the new name https://github.com/huggingface/diffusers/pull/3021/commits/9160e517b3bf8ee0b6ca4923c49c928f2a657ef9